<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calcuator</class>
 <widget class="QMainWindow" name="calcuator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>579</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>calcuator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/icon.png</normaloff>icon/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 33, 36);</string>
  </property>
  <widget class="QWidget" name="calcuator_2">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="2">
       <widget class="QPushButton" name="btn_9">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btn_sum">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
	background-color: rgb(255, 255, 127);
 border-radius:30px;
 color: rgb(0, 0, 0);
}
QPushButton:hover{
 color: rgb(0, 0, 0);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btn_sub">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
	background-color: rgb(255, 255, 127);
 border-radius:30px;
 color: rgb(0, 0, 0);
}
QPushButton:hover{
 color: rgb(0, 0, 0);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="btn_mul">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
	background-color: rgb(255, 255, 127);
 border-radius:30px;
 color: rgb(0, 0, 0);
}
QPushButton:hover{
 color: rgb(0, 0, 0);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="btn_3">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btn_8">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QPushButton" name="btn_cos">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>cos</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="btn_equal">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
	background-color: rgb(255, 255, 127);
 border-radius:30px;
 color: rgb(0, 0, 0);
}
QPushButton:hover{
 color: rgb(0, 0, 0);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4" rowspan="2">
       <widget class="QPushButton" name="btn_sqrt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>sqrt</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn_c">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
	background-color: rgb(255, 255, 127);
 border-radius:30px;
 color: rgb(0, 0, 0);
}
QPushButton:hover{
 color: rgb(0, 0, 0);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>c</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btn_5">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btn_4">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QPushButton" name="btn_0">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="btn_cot">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>cont</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="btn_1">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}


</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btn_6">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btn_7">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_sin">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>sin</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="btn_div">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
	background-color: rgb(255, 255, 127);
 border-radius:30px;
 color: rgb(0, 0, 0);
}
QPushButton:hover{
 color: rgb(0, 0, 0);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QPushButton" name="btn_dot">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
	background-color: rgb(255, 255, 127);
 border-radius:30px;
 color: rgb(0, 0, 0);
}
QPushButton:hover{
 color: rgb(0, 0, 0);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btn_tan">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}


</string>
        </property>
        <property name="text">
         <string>tan</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="btn_2">
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4" rowspan="3">
       <widget class="QPushButton" name="btn_log">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>162</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
 font: 600 28pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(63, 64, 66);
color: rgb(255, 255, 255);
 border-radius:30px;
}
QPushButton:hover{
 color:rgb(255,255,255);
 background-color:rgb(238,238,238);
}

</string>
        </property>
        <property name="text">
         <string>log</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLabel" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">font: 600 20pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
